tii1q_b1.py

```diff
@@ -1,17 +1,20 @@
 import pathlib
 
 from qibolab.channels import Channel, ChannelMap
-from qibolab.instruments.rfsoc import TII_RFSOC4x2
+from qibolab.instruments.rfsoc import RFSoC as TII_RFSOC4x2
```

Comment:
For the sake of simplicity, I think we should avoid the `TII_` prefix and just use `RFSoC` directly.